<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BackOffice NewWorld</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Utilisateur :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Utilisateur">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot de passe :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_MDP">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Connexion">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Deconnexion">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Deconnexion</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_Moderation">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Moderation</string>
     </attribute>
     <widget class="QTableView" name="tableView_Produit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>341</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_ProduitF">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_AccepterProduit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Accepter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_RefuserProduit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Refuser</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>NOUVELLE DEMANDE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Supprimer">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>330</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Modifier">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>225</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Vous avez </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Demande">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>demande en attente</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_mdr">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>440</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ajout Rayon</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>AJOUT RAYON : </string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_Rayon">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>341</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>44</y>
        <width>21</width>
        <height>71</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>44</y>
        <width>31</width>
        <height>71</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>13</y>
        <width>671</width>
        <height>51</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>671</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RAYON DEJA EXISTANT : </string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>490</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>NOUVEAU RAYON :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_AjoutRayon"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_AjoutRayon">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ajout Type de Produit</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>60</y>
        <width>210</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>NOUVEAU TYPE DE PRODUIT :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_AjoutTypeDeProduit">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>60</y>
        <width>262</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>AJOUT TYPE DE PRODUIT : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_AjoutTypeDeRayon_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>100</y>
        <width>85</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_TypeDeProduit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>341</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>146</y>
        <width>691</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>50</y>
        <width>31</width>
        <height>101</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>50</y>
        <width>21</width>
        <height>101</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>29</y>
        <width>691</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TYPE DE PRODUIT DEJA EXISTANT :</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>331</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>RAYON : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_AjoutTypeDeRayon"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_Fermer"/>
   </widget>
   <addaction name="menu_Fichier"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Fermer">
   <property name="text">
    <string>&amp;Fermer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
